src/pages/Assignments/SubmittedContent.css

```
@@ -1,91 +1,247 @@
+.submitted-content-table td {
+  color: var(--submitted-text);
+}
+
+.submitted-content-table tbody tr:last-child td {
+  border-bottom: 0;
+}
+
+.submitted-content-link {
+  color: #0d6efd;
+  text-decoration: none;
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)